model/src/main/kotlin/PackageCuration.kt

```diff
@@ -58,12 +57,7 @@ data class PackageCuration(
      * package with the given [identifier][pkgId].
      */
     private fun isApplicableIvyVersion(pkgId: Identifier) =
-        try {
-            val pkgIvyVersion = Semver(pkgId.version, Semver.SemverType.IVY)
-            pkgIvyVersion.satisfies(id.version)
-        } catch (e: SemverException) {
-            false
-        }
+        runCatching { Requirement.buildIvy(id.version).isSatisfiedBy(pkgId.version) }.getOrDefault(false)
```

Comment:
Did you intend to do this...? `runCatching { Requirement.buildIvy(id.version) }.isSatisfiedBy(pkgId.version).getOrDefault(false)`

Reply:
Sorry, I believe it should be like

```
val ivy = Requirement.buildIvy(id.version)

return runCatching { ivy.isSatisfiedBy(pkgId.version) }.getOrDefault(false)

```

Reply:
> Did you intend to do this...? runCatching { Requirement.buildIvy(id.version) }.isSatisfiedBy(pkgId.version).getOrDefault(false)

Not quite, it does not work that way as `runCatching` return a `Result` object. But see above, I had

    runCatching { Requirement.buildIvy(id.version) }.getOrNull()?.isSatisfiedBy(pkgId.version) ?: false

> Sorry, I believe it should be like

But that does not catch an exception in `buildIvy`... like said, a `SemverException` can be thrown in both `buildIvy()` (when the version range in the curation is not Ivy-compatible) and in `isSatisfiedBy()` (if the package version does not parse as a Semver), and I believe we want to catch both cases here, and return `false` in that case, just like we did before my syntax change.

Reply:
Oh I see, I fully misunderstood the intention of the commit. 

What I wrote was really out of scope then. So let's forget about it.

Note: the logic I put above I still believe makes sense to this commit you had pushed here yesteray for a short amount of time. (I assume it is what you are planning to do next)